lib/css/res.scss

```diff
@@ -43,6 +43,9 @@
 
 body.res-console-open {
 	overflow: hidden;
+	&.front-page.res-hide-tagline-frontpage .tagline {
```

Comment:
Looks like a merge conflict resolution issue here, this probably shouldn't be inside the console open block.